R/DatabaseConnector.R

```diff
@@ -460,6 +460,35 @@ connect <- function(connectionDetails,
     attr(connection, "dbms") <- dbms
     return(connection)
   }
+  if (dbms == "impala") {
+    writeLines("Connecting using Impala driver")
+    impalaDriverDirectory <- getOption("impalaDriverDirectory")
```

Comment:
@tomwhite - Should gently fail if `impalaDriverDirectory` and the requisite `jar` files do not exist, perhaps even providing users with instructions on how to install `Impala`